src/Symfony/Component/Form/ChoiceList/ArrayChoiceList.php

```diff
@@ -233,13 +233,13 @@ private function castableToString(array $choices, array &$cache = array())
                 }
 
                 continue;
-            } elseif (!is_scalar($choice)) {
+            } elseif (!is_scalar($choice) && null !== $choice) {
```

Comment:
@Tobion thanks, got it, so there is no need to use `~` for `null` value, if I revert the condition above as it will always use numeric values, is it ?